### ISSUE TITLE
Support aliases for patterns

### ISSUE THREAD

**Issue by spscally** [EngineHub/WorldEdit#2654]
### The Problem

Let's say I am working on a town and I use the following pattern for all of the roads: `55%gravel,25%tuff,20%coarse_dirt`. When adding new road, I have to make sure to either write down or remember what this pattern was for consistency.

### A Solution

Would it be possible to save (locally to clients) aliases for patterns? Then I could do something like:
1. `//alias mountain_town_road 55%gravel,25%tuff,20%coarse_dirt` - to save the alias
2. `//set mountain_town_road` - example of using it

### Alternatives

My current solution is to leave a sign near the build I'm working on that lists out any patterns I am using. This isn't perfect because the sign may be broken or forgotten about.

### Anything Else?

Apologies if this functionality already exists! If it doesn't and you all are willing to add it, I would be willing to implement it.

**Comment by wizjany** [comment]
> Would it be possible to save (locally to clients)

No, worldedit is a server-side mod. There used to be macro mods that allowed client-side keybind and command macros, perhaps look into something like that if they exist these days, this is generally outside the scope of worldedit.

**Comment by spscally** [comment]
> > Would it be possible to save (locally to clients)
> 
> No, worldedit is a server-side mod. There used to be macro mods that allowed client-side keybind and command macros, perhaps look into something like that if they exist these days, this is generally outside the scope of worldedit.

Oh right duh, it's not client-side. What do you think about adding server-side aliases then? It that something you would also consider to be out of the scope of worldedit?

If not, server-side aliases could be an interesting way for people collaborating on a build to share those patterns.
